tests/contrib/aredis/test_aredis.py

```diff
@@ -109,7 +109,7 @@ async def test_meta_override(tracer, test_spans):
     test_spans.assert_trace_count(1)
     test_spans.assert_span_count(1)
     assert test_spans.spans[0].service == "redis"
-    assert "cheese" in test_spans.spans[0].meta and test_spans.spans[0].meta["cheese"] == "camembert"
+    assert "cheese" in test_spans.spans[0].get_tags() and test_spans.spans[0].get_tag("cheese") == "camembert"
```

Comment:
```suggestion
    assert test_spans.spans[0].get_tag("cheese") == "camembert"
```

Reply:
yeah, probably better to not make this or the suggestion I made in `bottle/test.py` since that is how the tests were before. we shouldn't make any structural changes in this PR.